src/emqtt_cli.erl

```diff
@@ -152,19 +160,19 @@ main(PubSub, Opts0) ->
               end,
     case ConnRet of
         {ok, Properties} ->
-            io:format("Client ~s sent CONNECT~n", [get_value(clientid, NOpts)]),
+            log("Sent CONNECT~n", []),
             case PubSub of
                 pub ->
                     publish(Client, NOpts, proplists:get_value(repeat, Opts)),
-                    disconnect(Client, NOpts);
+                    disconnect(Client);
                 sub ->
                     subscribe(Client, NOpts),
                     KeepAlive = maps:get('Server-Keep-Alive', Properties, get_value(keepalive, NOpts)) * 1000,
                     timer:send_interval(KeepAlive, ping),
                     receive_loop(Client, Print)
             end;
         {error, Reason} ->
-            io:format("Client ~s failed to sent CONNECT due to ~p~n", [get_value(clientid, NOpts), Reason])
+            log_halt("Failed to send CONNECT due to: ~p~n", [Reason])
```

Comment:
Do we intentionally change behavior by adding an immediate halt after printing a message?

Reply:
Checked again all the `log_halt` calls:
- for publish command, there is nothing more to do anyway.
- for subscribe, we only support subscribing one topic, i don't think it's useful to keep the script running after failing to subscribe the only topic
- for disconnect, it's already on the way to halt

I have also added a `connect` command to support testing CONNECT packet only.